cmd/resource-topology-exporter/main.go

```diff
@@ -123,13 +125,17 @@ func parseArgs(args ...string) (ProgArgs, error) {
 
 	flags.BoolVar(&pArgs.RTE.Debug, "debug", false, " Enable debug output.")
 	flags.StringVar(&pArgs.RTE.TopologyManagerPolicy, "topology-manager-policy", defaultTopologyManagerPolicy(), "Explicitly set the topology manager policy instead of reading from the kubelet.")
+	flags.StringVar(&pArgs.RTE.TopologyManagerScope, "topology-manager-scope", defaultTopologyManagerScope(), "Explicitly set the topology manager scope instead of reading from the kubelet.")
```

Comment:
I would set the default to ```container``` which is the actual default https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-scopes

Reply:
let's re-evaluate this change later